tapir/coop/views/shareowner.py

```diff
@@ -595,6 +597,20 @@ class Meta:
             "is_company",
         ]
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # initiate after database has been initiated
```

Comment:
What has to be initiated here? Maybe this code code could be part of another method (that would have a name describing what is initiated).

Reply:
__init__ initiates an object of that class. It is necessary in that case, because otherwise Django would initiate the choices  for ChoiceFilter during runtime and throw an error, because the database wouldn't been initiated yet.

Reply:
But I also think it is a bit ugly tbh